modules/ROOT/pages/_partials/showcase-apps/security.adoc

```diff
@@ -0,0 +1,20 @@
+= Security Services
+
+== {device-security-service}
+
+
+From the showcase app:
+
+. Press the *Security* menu item.
```

Comment:
I'd shorten the sentance (and apply this to the other sentances too) to :

" Click *Security* "

Reply:
I was matching the Auth section
![image](https://user-images.githubusercontent.com/16667688/57127651-072f5380-6d89-11e9-8b53-b92e906820bf.png)

No issue to changing , but would use press as it is a mobile app and click wouldn't apply.
```
. Press *Security*
. From the drop down press *Device Security*
```

Reply:
usage is 
https://user-images.githubusercontent.com/16667688/56582990-7611ed00-65d0-11e9-8e60-23d7692c9c56.gif

Reply:
I think 'Press' is good for highlighting the mobile UI, but haven't researched if there are style issues with that.